src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2454,6 +2460,285 @@ static bool forwardsPending(UpdateContextResponse* upcrsP)
   return false;
 }
 
+/* ****************************************************************************
+*
+* updateEntity -
+*/
+static void updateEntity
```

Comment:
This big bunch of code was before in processContextElement() function. Thus, a side advantage of this PR is to get a better modularization.

NTC (just informative)